### PR TITLE
Some bugfixes

### PR DESCRIPTION
**Pull request by liyangguang**
- Events in chromedash-feature was not bubbled up correctly. Thus event handlers in features-page.js were not triggered.
- When applying filters, props inside chromedash-feature was not updated.

### CODE REVIEW

**Comment by liyangguang** [comment]
@robdodson

File: static/elements/chromedash-feature.js [hunk 1]
```diff
@@ -429,7 +429,7 @@ class ChromedashFeature extends LitElement {
               ${this._hasSampleLinks ? html`
                 <div class="sample_links">
                   <chromedash-multi-links title="Sample"
-                      .links="${this.feature.resources.samples}"
+                      .links="${this.feature.resources.samples || []}"
```
[the remark this answers is not in the document]
Comment (liyangguang) [reply static/elements/chromedash-feature.js]: You're right. I saw the error message but didn't look closely. The real issue is "hasSampleLinks" is not updated when filtering happens. Updated that.

File: static/elements/chromedash-featurelist.js [hunk 2]
```diff
@@ -73,6 +73,10 @@ class ChromedashFeaturelist extends LitElement {
     this.dispatchEvent(event);
   }
 
+  _bubbleUpEvent(e) {
```
[the remark this answers is not in the document]
Comment (liyangguang) [reply static/elements/chromedash-featurelist.js]: I tried `bubbles: true` but didn't work. Turns out I missed `composed: true`. (found this in your blog https://robdodson.me/interoperable-custom-elements/ haha). Updated.

**Comment by robdodson** [comment]
@jpmedley would you mind merging when you have a chance?

**Pull request merged**